packages/control/algorithm/surplus_controlled.py

```diff
@@ -2,13 +2,15 @@
 from typing import List, Optional, Tuple
 
 from control import data
+from control import state_machine
 from control.algorithm import common
 from control.loadmanagement import LimitingValue, Loadmanagement
 from control.counter import Counter
 from control.chargepoint import Chargepoint
 from control.algorithm.filter_chargepoints import (get_chargepoints_by_mode, get_chargepoints_by_mode_and_counter,
                                                    get_preferenced_chargepoint_charging, get_chargepoints_pv_charging,
                                                    get_chargepoints_surplus_controlled)
+from control.state_machine import StateMachine
```

Comment:
`StateMachine` wurde bereits in Zeile 5 als `state_machine.StateMachine` importiert.